components/engine_rocks/src/write_batch.rs

```diff
@@ -152,26 +229,28 @@ mod tests {
         let opt = RawDBOptions::default();
         opt.enable_unordered_write(false);
         opt.enable_pipelined_write(false);
-        opt.enable_pipelined_commit(true);
+        opt.enable_multi_batch_write(true);
```

Comment:
Add a test case for adding a lot of writes to one batch without enabling multi-batch-write.